units/provisioning_setup_host.go

```diff
@@ -258,7 +249,7 @@ func (j *setupHostJob) runHostSetup(ctx context.Context, targetHost *host.Host,
 // copyScript writes a given script as file "name" to the target host. This works
 // by creating a local copy of the script on the runner's machine, scping it over
 // then removing the local copy.
-func (j *setupHostJob) copyScript(ctx context.Context, settings *evergreen.Settings, target *host.Host, name, script string) error {
+func (j *setupHostJob) copyScript(ctx context.Context, settings *evergreen.Settings, target *host.Host, name, remoteName, script string) error {
```

Comment:
What's the reason for keeping the old setup script name around?

Reply:
I removed the change to. setupHostJob.